data/free worlds intro.txt

```diff
@@ -1049,6 +1117,7 @@ mission "FW Katya 5C"
 	
 	on complete
 		payment 100000
+		log `We brought Brother Greg from Harmony to Greenrock to find information on who could have done the bombing and where they would have tested it. I never would have guess that monks and pirates would have such a good relationship with eachother.`
```

Comment:
I would suggest some kind of introduction explaining who Brother Greg is (a monk).
> We brought a monk called Brother Greg ...